CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change log for Microsoft365DSC
 
 # UNRELEASED
+* AADGroup
```

Comment:
Can we please put the entry in the right location (alpha sorted) and at least have an empty line between header and next entry